spec.html

```diff
@@ -25491,6 +25491,16 @@ <h1>Object.defineProperty ( _O_, _P_, _Attributes_ )</h1>
         </emu-alg>
       </emu-clause>
 
+      <emu-clause id="sec-object.hasown">
+        <h1>Object.hasOwn ( _O_, _P_ )</h1>
```

Comment:
i think this needs to be alphabetized - between Object.getPrototypeOf and Object.is